Radio.m

```diff
@@ -3274,7 +3506,10 @@ - (void)socket:(GCDAsyncSocket *)sock didReadData:(NSData *)data withTag:(long)t
         [scan scanUpToString:@"\n" intoString:&payload];
 
         if (self.logRadioMessages)
-            NSLog(@"Data received - %@\n", payload);
+            // don't show messages of the form "R<seqNumber>|0|" (makes debugging easier)
```

Comment:
Actually not sure I agree...  there is an issue in the radio right now where a 0 response doesn't mean that stuff actually worked.  We can leave this in for now as I think the levels associated with LumberJack can be used to take care of this so we have our cake and eat it too.